{{cookiecutter.bot_project_name}}/Dockerfile

```diff
@@ -16,19 +16,24 @@ ARG GIT_HOST=""
 ARG GIT_PASSWORD=${CI_JOB_TOKEN}
 ARG GIT_LOGIN="gitlab-ci-token"
 
+RUN apt-get update && \
+    apt-get install -y sudo && \
+    echo "${NON_ROOT_USER} ALL = NOPASSWD: /usr/sbin/update-ca-certificates" > /etc/sudoers.d/express_bot && \
+    apt-get clean autoclean && \
+    apt-get autoremove --yes && \
+    rm -rf /var/lib/{apt,dpkg,cache,log}/
+
 # Poetry can't read password to download private repos
-RUN echo -e "machine ${GIT_HOST}\nlogin ${GIT_LOGIN}\npassword ${GIT_PASSWORD}" > ~/.netrc && \
-    apt-get update && \
-    apt-get install -y git sudo && \
+RUN apt-get install -y git && \
```

Comment:
Suggest to move `apt-get install -y git` into first RUN and remove clearning apt cache from second RUN.

Reply:
git is not used in first run, does it make sense to move it there?

Rmove cleaning apt cache - you are about 
`apt-get clean autoclean && \
 apt-get autoremove --yes && \
 rm -rf /var/lib/{apt,dpkg,cache,log}/ && \
 ` ?

Reply:
> git is not used in first run, does it make sense to move it there?

I think it just looks more clear, when all packages are installed at once.

> you are about apt-get clean autoclean && \ apt-get autoremove --yes && \ rm -rf /var/lib/{apt,dpkg,cache,log}/ && \

Yes.

Reply:
fixed